drivers/snowflake/snowflake.go

```diff
@@ -4,26 +4,48 @@
 package snowflake
 
 import (
+	"io"
 	"io/ioutil"
 	"strconv"
 
 	"github.com/sirupsen/logrus"
 	"github.com/snowflakedb/gosnowflake" // DRIVER: snowflake
+	"github.com/xo/tblfmt"
 	"github.com/xo/usql/drivers"
+	"github.com/xo/usql/drivers/metadata"
+	infos "github.com/xo/usql/drivers/metadata/informationschema"
+	"github.com/xo/usql/env"
 )
 
 func init() {
 	r := logrus.New()
 	r.Out, r.Level = ioutil.Discard, logrus.PanicLevel
 	var l gosnowflake.SFLogger = &logger{r}
 	gosnowflake.SetLogger(&l)
+	newReader := infos.New(
+		infos.WithPlaceholder(func(int) string { return "?" }),
+		infos.WithCustomColumns(map[infos.ColumnName]string{
+			infos.SequenceColumnsIncrement: "NULL",
```

Comment:
I don't think we can use null here, it'll fail to be scanned into a regular non-pointer string.